index.js

```diff
@@ -37,11 +37,27 @@ function handleError(error) {
   process.exitCode = 1;
 }
 
+// The following code block overrides commander's outputHelp()
```

Comment:
It would be better if the comment mentions the _why_ (the reason for overriding) as well.

You should mention that we want to customize the help message with MarkBind's header, but commander.js doesn't provide any API to customize the help message.

```js
// We want to customize the help message to print MarkBind's header,
// but commander.js does not provide an API directly for doing so.
// Hence we override commander's outputHelp() completely.
```